server/dashboard.js

```diff
@@ -472,10 +465,7 @@ const validateEditClient = [
     .trim()
     .notEmpty(),
   Validator.body(['reminderTimeout', 'fallbackTimeout']).trim().isInt({ min: 0 }),
-  Validator.oneOf([
-    Validator.body(['responderPhoneNumbers']).trim().notEmpty(),
-    Validator.body(['alertApiKey', 'responderPushId']).trim().notEmpty(),
-  ]),
+  Validator.body(['responderPhoneNumbers']).trim().notEmpty(),
```

Comment:
Same here about being able to move this up into the other `trim().notEmpty()` validator.